src/index.ts

```diff
@@ -12,17 +12,21 @@ const createJWKSMock = (
     ...keypair,
     jwksOrigin: jwksBase,
   })
-  const server = setupServer(
-    rest.get(new URL(jwksPath, jwksBase).href, (_, res, ctx) =>
-      res(ctx.status(200), ctx.json(JWKS))
-    )
+
+  const handler: HttpHandler = http.get(new URL(jwksPath, jwksBase).href, () =>
+    HttpResponse.json(JWKS)
   )
 
   const kid = () => JWKS.keys[0].kid
 
+  let server: SetupServerApi
```

Comment:
I do not like this schmutzige mutation. However currently it is necessary to keep backward compatibility for the `stop` method.